sql/src/main/java/io/crate/execution/engine/collect/collectors/LuceneOrderedDocCollector.java

```diff
@@ -168,7 +160,9 @@ public void close() {
             collector = new MinimumScoreCollector(collector, minScore);
         }
         searcher.search(query, collector);
-        return scoreDocToIterable(topFieldCollector.topDocs().scoreDocs);
+        ScoreDoc[] scoreDocs = topFieldCollector.topDocs().scoreDocs;
+        TopFieldCollector.populateScores(scoreDocs, searcher, query);
```

Comment:
Does this mean we're now always scoring even if the former `doDocsScores` flag would be false?

Reply:
Well done, thanks for catching this. This was my mistake, since the boolean trackDocScores was removed from TopFieldCollector#create and then additionally TopFieldCollector#populateScores introduced.

Reply:
Since the refactoring was done in multiple steps i first removed it, since TopFieldCollector#create wasn't using it anymore and then removed it also from LuceneOrderedDocCollector.

Reply:
I fixed it, and added a test to LuceneOrderedDocCollectorTest to cover the doDocsScores flag now.